src/software/SfM/main_ComputeFeatures.cpp

```diff
@@ -231,9 +243,22 @@ int main(int argc, char **argv)
         if (!ReadImage(sView_filename.c_str(), &imageGray))
           continue;
 
+        Image<unsigned char> * mask = nullptr;
+
+        const std::string sImageMask_filename = stlplus::create_filespec(sfm_data.s_root_path,
+         stlplus::basename_part(sView_filename), "_mask.png");
+
+        if(bUseMask && stlplus::file_exists(sImageMask_filename))
```

Comment:
Global mask does not have the priority here (perhaps we can change the test to)
if bUseMask && globalMask.Width() > 0 && file exists...